Routes/api-router.js

```diff
@@ -1,5 +1,7 @@
 const apiRouter = require("express").Router();
```

Comment:
The folder name here `Routes` still has a capital letter...

Reply:
This has been changed